raster.py

```diff
@@ -272,6 +276,56 @@ def get_data_at_coords(dataset: AmazonGeoTiff, x: float, y: float, month: int) -
   else:
     return value
 
+def get_predictions_at_each_pixel(
+    model: tf.keras.Model,
+    feature_transformer: ColumnTransformer,
+    geotiffs: dict[str, AmazonGeoTiff],
+    bounds: Bounds):
```

Comment:
Could we please add a method description here and specify in it what these parameters are?

Reply:
Did we resolve this?